Bottom: 6b2425e4bd9a86680ccf753b632866ed06b98e43
Top:    41cce43f059f01becac52b9d0535bf54903736b4
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-20 15:16:02 -0500

Refresh of playbook-add-docker_health-module

---
diff --git a/ansible-playbook/library/docker_health.py b/ansible-playbook/library/docker_health.py
index 57cf6357..c9622c3b 100644
--- a/ansible-playbook/library/docker_health.py
+++ b/ansible-playbook/library/docker_health.py
@@ -53,9 +53,9 @@ EXAMPLES = '''
   docker_health:
     name: mycontainer
     register: health
-    retries: 10
-    delay: 3
-    until: '{{ health.Heath.Status }} is "healthy"'
+    retries: 5
+    delay: 30
+    until: health.Health.Status == "healthy"
 '''
 
 RETURN = '''
